modules/oc-mirror-installing-plugin.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="installation-oc-mirror-installing-plugin_{context}"]
+= Installing the oc-mirror OpenShift CLI plug-in
+
+// TODO: intro
+
+// You can download and install the oc-mirror CLI plug-in to... TODO
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Download the oc-mirror CLI plug-in.
+
+.. Navigate to the link:https://console.redhat.com/openshift/downloads[Downloads] page of the {console-redhat-com}.
+
+.. Under the *Command-line interface (CLI) tools* section, click *Download* for *TODO*.
```

Comment:
@afflom @jpower432 Do we happen to know what the entry for oc-mirror will be called yet for the downloads page in cluster manager?

Reply:
It will likely be called "oc mirror plugin"

Reply:
Thanks, I'll go with this for now, with a note to confirm later